source/features/open-all-notifications.css

```
@@ -3,3 +3,9 @@
 	~ .rgh-open-notifications-button {
 	display: inline-block !important;
 }
+
+.js-notifications-mark-selected-actions
+	:not(.mr-2)
+	+ .rgh-open-selected-button {
+	margin-left: var(--base-size-8, 8px) !important;
```

```suggestion
	margin-left: var(--base-size-8, 80px) !important;
```

We want breakages to be visible